src/app/app-modules/pharmacist/worklist/worklist.component.html

```
@@ -1,229 +1,112 @@
-        class="btn btn-link pull-right">
-        {{ currentLanguageSet?.bendetails?.emergencyCaseSheet }}
-      </button>
-      <button
-        mat-raised-button
-        class="pull-right mat_blue"
-        color="primary"
-        id="refresh"
-        (click)="loadPharmaWorklist()">
-        {{ currentLanguageSet?.common?.refresh }}
-      </button>
+<div class="worklist">
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)